examples/03_connectivity/plot_seed_based_connectome.py

```diff
@@ -1,57 +1,89 @@
 """
-Extract signals and plot a connectome for seed-based atlases
-============================================================
+Extract signals and plot a connectome for atlas-based spheres
```

Comment:
I don't really understand what "atlas-based spheres" means.

I would suggest the following title: "Extract signals from spheres and plot a connectome"